lib/manifestTools/manifestValidator.js

```diff
@@ -131,8 +134,17 @@ function applyValidationRules(w3cManifestInfo, platformModules, platforms) {
     });
 
     return Q.allSettled(platformTasks);
-  })
-  .thenResolve(allResults);  
+  }
+  
+  // Don't run the "All Platform" validattion for Edge Extensions since they are not w3c compliant
+  if (platforms.length === 1 && platforms[0] === constants.EDGE_EXTENSION_FORMAT) {

```

Comment:
If there are other platforms that might want to use similar logic to bypass the standard validation and only do platform validation we could make this a global parameter that gets passed into the CLI